tests/unit/test_dataset.py

```diff
@@ -72,7 +72,7 @@ def ner_utils_test_data(scope="module"):
 
 def test_maybe_download():
     # ToDo: Change this url when repo goes public.
-    file_url = "https://raw.githubusercontent.com/Microsoft/Recommenders/" "master/LICENSE"
+    file_url = "https://github.com/microsoft/nlp/blob/master/LICENSE"
```

Comment:
I think this would give you an error, you should link what I added before or `https://raw.githubusercontent.com/microsoft/nlp/master/LICENSE`

Reply:
Thanks! I just changed it back to what it was before. 
I tried both https://raw.githubusercontent.com/microsoft/nlp/master/LICENSE and https://github.com/microsoft/nlp/raw/master/LICENSE, but neither worked, which is strange.